package CH4;

import java.util.Scanner;

import org.junit.Test;

public class TestException {
	@Test
	public void test() {
		Scanner scan = new Scanner(System.in);
		int i = scan.nextInt();
		System.out.println(i);

	}

}
